### PR TITLE
Fixing linting after it was added to the project

### PR DESCRIPTION
**Pull request by brunocarneiro**
- Fixing linting errors after Linting PR was approved.

### CODE REVIEW

File: src/services/anchorService.js [hunk 1]
```diff
@@ -1,19 +0,0 @@
-/**
```
Comment (brunocarneiro) [inline src/services/anchorService.js]: @jpsantosbh Is this class need? It has the same interface as the anchor service implementation

Comment (jpsantosbh) [reply src/services/anchorService.js]: This is a wrapper class that hosts the impl. If every client code is getting the IMPL from the container and not via require(). We don't need it anymore.

**Comment by dmelosantos** [comment]
@brunocarneiro this PR is stale, shouldn't we close it?

**Comment by brunocarneiro** [comment]
This PR got old. If we feel this is still needed will open it again.

**Pull request closed without merging**